neqo-qpack/src/encoder.rs

```diff
@@ -146,6 +146,10 @@ impl QPackEncoder {
     fn recalculate_blocked_streams(&mut self) {
         let acked_inserts_cnt = self.table.get_acked_inserts_cnt();
         self.blocked_stream_cnt = 0;
+        #[expect(
+            clippy::iter_over_hash_type,
+            reason = "OK to loop over unACKed blocks in an undefined order."
```

Comment:
Is it though? Would an ordered iteration have advantages? (Also applies to the next two instances of this in this file.)

Reply:
Are you suggesting that maybe we could get more efficiency from these if we knew what was most active?  I suspect that headers would be more likely to be changing for more recent streams, but I don't know how to avoid iterating the whole collection for these functions.  We'd need a completely different structure, I guess.  One that only stored blocked streams, in this case.  That might be faster.

What's the data structure?  `BinaryHeap`?

Reply:
You thought about this more than I did - it just felt to me that this was the one suppression I added where maybe something else might be done. We have a long-standing bug (#49) on optimizing QPACK, any changes can be done as part of that work if we ever get to it.